src/test/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import core.basesyntax.Machine;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("core.basesyntax.Bulldozer has started working.");
```

Comment:
same for all messages

```suggestion
        System.out.println("Bulldozer has started working.");
```

Reply:
not fixed (fix it everywhere)